tests/test_merge.py

```diff
@@ -72,15 +72,10 @@ def make_commit(name, skip=False):
     return sha
 
 
-# For all test_merge_branch tests below, we cannot assert commits in the log line-by-line because the ordering of the
-# commit messages is not guaranteed because the timestamps on the commits are all the same. Adding sleep statements
-# allows the order to be guaranteed and repeatable but will slow down the tests.
```

Comment:
Removed this comment since it doesn't seem to be applicable anymore (there are no use of sleep statements in the below tests)